qrenderdoc/Windows/Dialogs/LiveCapture.ui

```diff
@@ -280,10 +287,10 @@
            </property>
           </spacer>
          </item>
-         <item row="1" column="0" colspan="2">
-          <widget class="QPushButton" name="triggerImmediateCapture">
+         <item row="6" column="0">
+          <widget class="QPushButton" name="changeContext">
            <property name="text">
-            <string>Capture Frame(s) Immediately</string>
+            <string>Change Context</string>
```

Comment:
"Change context" is not a good name for this here - 'context' is a very GL-specific term, which may not even be intuitive to users who aren't using GL directly. E.g. in both your examples this is called a _surface_ rather than a context. Changing the name also side-steps problems with GL where if the same context is used for multiple surfaces, we're not actually changing context at all.

I don't think there's a single word that will make sense in all cases, but 'window' is probably the best compromise. It makes sense for all desktop platforms, and on android when an application does have multiple surfaces, the idea of changing the active window should have a clear parallel to surfaces.

I also think it would be better to call the operation 'cycle' since the action is just to select the next item in the list, in a circle, rather than giving a full list and allowing arbitrary selection.

So I'd suggest changing this to "Cycle active window". This also means it aligns with the text that is displayed in the overlay.